app.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
+
+const books = JSON.parse(localStorage.getItem('books')) || [];
+const form = document.querySelector('#form1');
+let j = 1;
+const list = document.querySelector('#list');
+
+for (let i = 0; i < books.length; i += 1) {
+  const listItem = document.createElement('li');
+  listItem.innerHTML = `${books[i].title}<br> ${books[i].author}<br>`;
+  list.appendChild(listItem);
+
+  // Create a Remove button for each list item
+  const removeButton = document.createElement('button');
+  removeButton.textContent = 'Remove';
+  removeButton.addEventListener('click', () => {
+    // Remove the corresponding data from the formDataArray and local storage
+    books.splice(i, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    // Remove the list item from the HTML list
+    listItem.remove();
+  });
+  const hr = document.createElement('hr');
+  listItem.appendChild(removeButton);
+  listItem.appendChild(hr);
+  list.appendChild(listItem);
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const title = document.querySelector('#title');
+  const author = document.querySelector('#author');
+
+  const formData = {
+    id: j,
+    title: title.value,
+    author: author.value,
+  };
+  j += 1;
+  books.push(formData);
+  localStorage.setItem('books', JSON.stringify(books));
```

Comment:
- [ ] [OPTIONAl] You have done really well, I have some suggestions you can implement to improve this project first one is, why don't you check for empty title and author values before adding the books to the collection, as currently we can add empty books also, which is not a good UI practice.
- [ ] [OPTIONAL] I would also suggest to empty the form fields once the  add button is clicked, as currently if a user wants to add a second book he first has to clear the input fields and then write the new title and author, from UI perspective this is not great either, these small fixes can greatly improve the UI of the application :+1: